src/outfit.ts

```diff
@@ -58,15 +57,12 @@ const bestAdventuresFromPants =
 export function freeFightOutfit(requirement?: Requirement): void {
   const equipMode = myFamiliar() === $familiar`Machine Elf` ? "dmt" : "free";
   const bjornChoice = pickBjorn(equipMode);
-  const compiledRequirements = requirement ?? new Requirement([], {});
-  const compiledOptions = compiledRequirements.maximizeOptions;
-  const compiledParameters = compiledRequirements.maximizeParameters;
 
-  const forceEquip = compiledOptions.forceEquip ?? [];
-  const bonusEquip = compiledOptions.bonusEquip ?? new Map<Item, number>();
-  const preventEquip = compiledOptions.preventEquip ?? [];
-  const preventSlot = compiledOptions.preventSlot ?? [];
-  const parameters = compiledParameters;
+  const parameters = requirement?.maximizeParameters ?? [];
```

Comment:
We should make this a function that we add to libram; something that essentially explodes a Requirement object into a bunch of consts.

Reply:
after more thought i think maybe we should do something like your requirement builder thing